### ISSUE TITLE
Forward role attribute

### ISSUE THREAD

**Issue by openscript** [maxeth/react-type-animation#35]
Thank you for your work on this package!

I've followed your accessibility guide and added a `aria-label`. If we add an `aria-label` also a `role` should be added:

![image](https://github.com/maxeth/react-type-animation/assets/1105080/7c49cad1-f221-491f-964c-c0a252ce9022)

So I suggest to also accept a `role` prop and forward it to the created wrapping element.

**Comment by maxeth** [comment]
Hey,

I don't think setting an `aria-label` is the optimal solution, because it's intended to be used on **interactive** elements that are supposed to be accessibly via assistive technologies .

Since such an animation is for decorative purposes and has 0 interaction, I think the only proper role is `"presentation"`, and in that case, all ARIA semantics will be automatically removed from the element, including  `aria-label`, similarly to just setting a css display `none`.

At this point I'd just go with the [visually-hidden class](https://react-type-animation.vercel.app/accessibility#1st-approach-visually-hidden-class) if you want to get rid of the lighthouse warnings.

**Comment by maxeth** [comment]
Added in `react-type-animation@3.2.0`.
